packages/shared/lib/services/proxy.service.ts

```diff
@@ -341,7 +339,7 @@ class ProxyService {
             activityLogs.push({
                 level: 'error',
                 environment_id,
-                activity_log_id: activityLogId,
+                activity_log_id: activityLogId as number, // In DryRun this value can be empty
```

Comment:
would it make sense to actually ensure activityLogId is never empty? Probably related question: does the `id` field of activityLog need to be an optional?

Reply:
yep I agree with you, I initially did this fix. But it was a mess because there is a lot of place where activityLogId=undefined means we don't want to log on purpose, so others where it was forgotten (or at least it's not clear why it's not there), and this specific case where we want to log but not to the DB. 
The migration will hopefully clarify a bit this

Reply:
(Im re-reading myself) to be more clear, we don't have an `activityLogId` in dryRun, so if I want to make it required there, I need to "invent" one which could lead to other issues since other dev could assume we do have a real one